vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -370,7 +372,7 @@ func (d *VolumeDriver) Mount(r volume.MountRequest) volume.Response {
 	fstype := fs.FstypeDefault
 	isReadOnly := false
 	if err != nil {
-		_, err := d.decrRefCount(r.Name)
+		d.decrRefCount(r.Name)
```

Comment:
This Get() is in Mount() and needed to identify the access type for the volume. We should deprecate the access-type, there is no use case for it.